### PR TITLE
profile manifest for XCreds 5.0 (7176)

### PR DESCRIPTION
**Pull request by twocanoes**
profile manifest for XCreds 5.0 (7176)

### CODE REVIEW

**Comment by kevinmcox** [comment]
Nope, just need more time in the day.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 1]
```diff
@@ -1340,7 +1572,7 @@ changing “passwordID” to the correct element ID. If the value you typed into
 		</dict>
 		<dict>
 			<key>pfm_description</key>
-			<string>Name for Shares menu item. Default: "Shares".</string>
+			<string>Name for Shares menu item. Default: &quot;Shares&quot;.</string>
```
Comment (kevinmcox) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:1575]: Is this an intentional change, it displays fine as is in both ProfileCreator and Imazing Profile Editor.

**Comment by kevinmcox** [comment]
I notice the `pfm_app_min` key is not in use at all, which is what is used to specify the version of the app that started supporting a specific key.

Wouldn't we want to mark that all these new keys require Xcreds 5?

**Comment by relgit** [comment]
Thanks for the update @twocanoes!

This could be a good opportunity to also give nicer titles to `idpHostName`, `idpHostNames`, and `passwordElementID`. Would you like to do that?

**Review by relgit — APPROVED** [review]

**Pull request merged**